cloudformation/calculation-athena.yaml

```diff
@@ -52,11 +52,51 @@ Resources:
             Parameters:
               {
                 "ignore.malformed.json": true,
-                "serialiazation.format": 1,
+                "serialization.format": 1,
                 "field.delim": "",
               }
             SerializationLibrary: "org.openx.data.jsonserde.JsonSerDe"
 
+  RateTableTable:
+    Type: AWS::Glue::Table
+    Properties:
+      CatalogId: !Ref AWS::AccountId
+      DatabaseName: !Ref CalculationsDB
+      TableInput:
+        Name: btm_rate_tables
+        TableType: EXTERNAL_TABLE
+        Parameters:
+          has_encrypted_data: "true"
+          projection.enabled: "true"
+          projection.datetime.type: date
+          projection.datetime.range: "2022-01-01,NOW"
+          projection.datetime.format: "yyyy-MM-dd"
+          projection.datetime.interval: 1
+          projection.datetime.interval.unit: DAYS
+          storage.location.template:
+            !Join [
+              "",
+              ["s3://", !Ref ConfigBucket, "/rate_tables/prices.csv"],
+            ]
+        StorageDescriptor:
+          Columns:
+            - { Name: "supplier_client_id", Type: string }
+            - { Name: "supplier_transaction_type_id", Type: string }
+            - { Name: "volume_min", Type: int }
+            - { Name: "volume_max", Type: int }
+            - { Name: "transaction_price", Type: float }
+            - { Name: "start", Type: timestamp }
+          Compressed: true
+          InputFormat: "org.apache.hadoop.mapred.TextInputFormat"
+          Location: !Join ["", ["s3://", !Ref ConfigBucket, "/rate_tables/"]]
+          StoredAsSubDirectories: false
+          OutputFormat: "org.apache.hadoop.hive.ql.io.IgnoreKeyTextOutputFormat"
+          SerdeInfo:
+            Parameters:
+              field.delim: ','
+              skip.header.line.count: 1
+            SerializationLibrary: "org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe"
```

Comment:
There's a lot of unneeded quotation going on in our YAML files. I'm always trying to use as few quotes as possible to increase readability. If you do want to use quotes, then better use single quotes, as double quoted values are possibly interpolated by the YAML engine, whereas single quotes are always literal strings.